packages/cli/commands/preview.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const path = require('path');
+const { i18n } = require('../lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const { getAccountId } = require('../lib/commonOpts');
+const { EXIT_CODES } = require('../lib/enums/exitCodes');
+const { getCwd } = require('@hubspot/cli-lib/path');
+const { preview } = require('@hubspot/cli-lib/lib/preview');
+const { getUploadableFileList } = require('../lib/upload');
+const { trackCommandUsage } = require('../lib/usageTracking');
+
+const i18nKey = 'cli.commands.preview';
+
+exports.command = 'preview <src> <dest>';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  const { src, dest, notify, initialUpload } = options;
+
+  const accountId = getAccountId(options);
+  const absoluteSrc = path.resolve(getCwd(), src);
+
+  if (!validateSrcPath(absoluteSrc)) {
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  const filePaths = await getUploadableFileList(absoluteSrc, false);
+  trackCommandUsage('preview', accountId);
+  preview(accountId, absoluteSrc, dest, {
+    notify,
+    filePaths,
+    initialUpload,
+  });
+};
+
+exports.builder = yargs => {
+  yargs.positional('src', {
+    describe: i18n(`${i18nKey}.positionals.src.describe`),
+    type: 'string',
+  });
+  yargs.positional('dest', {
+    describe: i18n(`${i18nKey}.positionals.dest.describe`),
+    type: 'string',
+  });
+  yargs.option('initial-upload', {
+    alias: 'i',
+    describe: i18n(`${i18nKey}.options.initialUpload.describe`),
+    type: 'boolean',
+  });
+  return yargs;
+};
+
+const validateSrcPath = src => {
+  const logInvalidPath = () => {
+    logger.log(
+      i18n(`${i18nKey}.errors.invalidPath`, {
+        path: src,
+      })
```

Comment:
Should we consider making this `logger.error` instead of the standard `log` since that will help stylize the output for this invalid path error?